server/tests/test.js

```diff
@@ -0,0 +1,130 @@
+import 'babel-polyfill'
+import { product, adminData, userData } from '../tests/constants'
+
+const { MongoMemoryServer } = require('mongodb-memory-server')
+const prepareApp = require('../app')
+const request = require('supertest')
+const mongoose = require('mongoose')
+const Cart = require('../models/Cart')
+const Product = require('../models/Product')
+
+describe('Test', function () {
+  let mongoServer, app
+
+  beforeAll(async () => {
+    mongoServer = new MongoMemoryServer()
+    const mongoUri = await mongoServer.getUri()
+
+    const server = await prepareApp(mongoUri)
+    app = request(server)
+  })
+
+  test('Create new user, admin', async () => {
```

Comment:
@AStashevskaya Could you please create test case with creating `user` with incorrect email, like `test@`